draft-ietf-rats-architecture.md

```diff
@@ -226,9 +227,9 @@ Relying Party:
 This is a generalization of the ML model use case above, where
 the data can be any highly confidential data, such as health data
 about customers, payroll data about employees, future business plans, etc.
-An assessment of system state is made against a set of policies
-to evaluate the state of a system using attestations for the system
-requesting data. Attestation is desired to prevent leaking data to
+As part of the attestation procedure, an assessment is made against a set
+of policies to evaluate the state of the system which is requesting
```

Comment:
```suggestion
of policies to evaluate the state of the system that is requesting
```
("that" and "which" are generally interchangeable in British English, but in American there is a subtle difference per Chicago Manual of Style.  The RFC Edit pass likes to correct these so pre-emptively doing so here.)